src/Symfony/Bundle/SecurityBundle/Resources/config/security_listeners.php

```diff
@@ -0,0 +1,285 @@
+            ])
+
+        ->set('security.authentication.basic_entry_point', BasicAuthenticationEntryPoint::class)
+
+        ->set('security.channel_listener', ChannelListener::class)
+            ->tag('monolog.logger', ['channel' => 'security'])
+            ->args([
+                service('security.access_map'),
+                service('security.authentication.retry_entry_point'),
+                service('logger')->nullOnInvalid(),
+            ])
+
+        ->set('security.access_map', AccessMap::class)
+
+        ->set('security.context_listener', ContextListener::class)
+            ->tag('monolog.logger', ['channel' => 'security'])
+            ->args([
+                service('security.untracked_token_storage'),
+                [],
+                abstract_arg('Provider Key'),
+                service('logger')->nullOnInvalid(),
+                service('event_dispatcher')->nullOnInvalid(),
+                service('security.authentication.trust_resolver'),
+            ])
+
+        ->set('security.logout_listener', LogoutListener::class)
+            ->abstract()
+            ->args([
+                service('security.token_storage'),
+                service('security.http_utils'),
+                abstract_arg('event dispatcher'),
+                [], // Options
+            ])
+
+        ->set('security.logout.listener.session', SessionLogoutListener::class)->abstract()
+
+        ->set('security.logout.listener.cookie_clearing', CookieClearingLogoutListener::class)->abstract()
+
+        ->set('security.logout.listener.default', DefaultLogoutListener::class)
+            ->abstract()
+            ->args([
+                service('security.http_utils'),
+                abstract_arg('target url'),
+            ])
+
+        ->set('security.authentication.form_entry_point', FormAuthenticationEntryPoint::class)
+            ->abstract()
+            ->args([
+                service('http_kernel'),
+            ])
+
+        ->set('security.authentication.listener.abstract')
+            ->abstract()
+            ->tag('monolog.logger', ['channel' => 'security'])
+            ->args([
+                service('security.token_storage'),
+                service('security.authentication.manager'),
+                service('security.authentication.session_strategy'),
+                service('security.http_utils'),
+                abstract_arg(''),
+                service('security.authentication.success_handler'),
+                service('security.authentication.failure_handler'),
+                [],
+                service('logger')->nullOnInvalid(),
+                service('event_dispatcher')->nullOnInvalid(),
+            ])
+
+        ->set('security.authentication.custom_success_handler', CustomAuthenticationSuccessHandler::class)
+            ->abstract()
+            ->args([
+                abstract_arg('The custom success handler service id'),
```

Comment:
"id" should be removed as it's not the id being injection but the service iteslf